config.yml.default

```diff
@@ -1,131 +1,131 @@
-url: "https://lichess.org/"  # Lichess base URL.
+token: "lip_yourtokenherexxxxx"    # Lichess OAuth2 Token.
+url: "https://lichess.org/"        # Lichess base URL.
 
-engine:                      # Engine settings.
-  dir: "./engines/"          # Directory containing the engine. This can be an absolute path or one relative to lichess-bot/.
-  name: "engine_name"        # Binary name of the engine to use.
-  working_dir: ""            # Directory where the chess engine will read and write files. If blank or missing, the current directory is used.
-                             # NOTE: If working_dir is set, the engine will look for files and directories relative to this directory, not where lichess-bot was launched. Absolute paths are unaffected.
-  protocol: "uci"            # "uci", "xboard" or "homemade"
-  ponder: true               # Think on opponent's time.
+engine:                            # Engine settings.
+  dir: "./engines/"                # Directory containing the engine. This can be an absolute path or one relative to lichess-bot/.
+  name: "engine_name"              # Binary name of the engine to use.
+  working_dir: ""                  # Directory where the chess engine will read and write files. If blank or missing, the current directory is used.
+                                   # NOTE: If working_dir is set, the engine will look for files and directories relative to this directory, not where lichess-bot was launched. Absolute paths are unaffected.
+  protocol: "uci"                  # "uci", "xboard" or "homemade"
+  ponder: true                     # Think on opponent's time.
 
   polyglot:
-    enabled: false           # Activate polyglot book.
+    enabled: false                 # Activate polyglot book.
     book:
-      standard:              # List of book file paths for variant standard.
+      standard:                    # List of book file paths for variant standard.
         - engines/book1.bin
         - engines/book2.bin
-#     atomic:                # List of book file paths for variant atomic.
+#     atomic:                      # List of book file paths for variant atomic.
 #       - engines/atomicbook1.bin
 #       - engines/atomicbook2.bin
 #     etc.
 #     Use the same pattern for 'giveaway' (antichess), 'crazyhouse', 'horde', 'kingofthehill', 'racingkings' and '3check' as well.
-    min_weight: 1            # Does not select moves with weight below min_weight (min 0, max: 65535).
-    selection: "weighted_random" # Move selection is one of "weighted_random", "uniform_random" or "best_move" (but not below the min_weight in the 2nd and 3rd case).
-    max_depth: 8             # How many moves from the start to take from the book.
+    min_weight: 1                  # Does not select moves with weight below min_weight (min 0, max: 65535).
+    selection: "best_move" # Move selection is one of "weighted_random", "uniform_random" or "best_move" (but not below the min_weight in the 2nd and 3rd case).
```

Comment:
`weighted_random` is better because it leads to more variety.

Reply:
Changed.